HidingNavigationBar/HidingNavigationBarManager.swift

```diff
@@ -56,6 +56,9 @@ public class HidingNavigationBarManager: NSObject, UIScrollViewDelegate, UIGestu
 		if viewController.navigationController == nil || viewController.navigationController?.navigationBar == nil {
 			fatalError("ViewController must be within a UINavigationController")
 		}
+        
+        viewController.extendedLayoutIncludesOpaqueBars = true
```

Comment:
This is a separate issue from ObjC compatibility, and should be split into it's own PR/issue.